cpp/include/cuml/fil/multi_reduction.h

```diff
@@ -0,0 +1,31 @@
+#pragma once
+#include <raft/cuda_utils.cuh>
+// data [T]ype, reduction [R]adix
```

Comment:
It's better to document template parameters together with the rest of the function documentation.